HelloWorld/app/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "net.boondockradio.helloworld"
+        minSdkVersion 16
+        targetSdkVersion 23
```

Comment:
This is the distribution of OS version in the world.
Most api level is over 16,so we support `minSdkVersion 16`.
![OS](https://chart.googleapis.com/chart?chs=500x250&cht=p&chco=c4df9b%2C6fad0c&chf=bg%2Cs%2C00000000&chd=t%3A0.1%2C2.0%2C1.9%2C18.9%2C31.6%2C35.4%2C10.1&chl=Froyo%7CGingerbread%7CIce%20Cream%20Sandwich%7CJelly%20Bean%7CKitKat%7CLollipop%7CMarshmallow)

https://developer.android.com/about/dashboards/index.html